extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -106,40 +106,40 @@ public function getNumberOfUserContribForArticle( $articleId, $userId ) {
 	 * @return mixed
 	 */
 	public function getWikiVariables() {
-		$wg = F::app()->wg;
+		global $wgSitename, $wgCacheBuster, $wgDBname, $wgDefaultSkin,
```

Comment:
I am curious, why are you using explicit globals here, how is it better than the previous solution?